### PR TITLE
OCPBUGS-5826: update PollConsoleUpdates to only check new manifests

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

**Comment by rhamilto** [comment]
@spadgett, mind reviewing https://github.com/openshift/console/pull/12462/commits/a25f6014f543800e93e6f49d073a16ff222e4585?

**Comment by spadgett** [comment]
FYI this might conflict with #12470

**Comment by rhamilto** [comment]
/hold

broken test

File: frontend/public/components/app.jsx [hunk 1]
```diff
@@ -374,25 +375,32 @@ const PollConsoleUpdates = React.memo(function PollConsoleUpdates() {
   const prevPluginManifestsData = prevPluginManifestsDataRef.current;
   const stateInitialized = _.isEmpty(updateError) && !_.isEmpty(prevUpdateData);
 
-  const pluginsListChanged = !_.isEmpty(_.xor(prevUpdateData?.plugins, updateData?.plugins));
+  const newPluginsList = _.xor(prevUpdateData?.plugins, updateData?.plugins);
```
Comment (spadgett) [inline frontend/public/components/app.jsx:378]: I think this will attempt to fetch the manifest for *removed* plugins as well, which we don't want.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**